config.js

```diff
@@ -476,6 +476,9 @@ var config = {
     // Hides add breakout room button
     // hideAddRoomButton: false,
 
+    // Disables join breakout room for non-moderator participants.
```

Comment:
Why not just  an option to hide the join button period? This way API users can decide. What if not all moderators should see it, but just the meeting "owner" ?

Reply:
It felt more consistent with the rest of the implementation. Maybe we could provide a distinct flag for hiding the button, but I'm not sure how to handle the owner case.

Reply:
How so? The `hideAddRoomButton` makes no distinction.